verification/curator-service/ui/src/components/FiltersModal.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useDebugValue } from 'react';
```

Comment:
`useDebugValue` should be removed as it is not used anywhere

Reply:
fixed